cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,136 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should have the ability to navigate pages', () => {
+    cy.addWorker(100);
```

Comment:
```suggestion
    cy.addWorker(6);
```
we do not need to add 100 workers here, your test runs a huge amount of times. After this change, pls be sure that your test works as expected